### ISSUE TITLE
Illegal characters and Windows path issues

### ISSUE THREAD

**Issue by Jeopardye** [electrode-io/electrode-native#510]
I'm running into the following issue when running 'ern create-container' on my Windows 10 machine:

Creating Container for ourAppReactNative:android:1.0.1 from Cauldron[buildAndroidPluginsViews] Something went wrong: Error: fatal: cannot create directory at 'plugins/ern_v0.10.2+/@apps:our_common_lib_v0.0.1+': Invalid argument

[fillContainerHull] Something went wrong: Error: fatal: cannot create directory at 'plugins/ern_v0.10.2+/@apps:our_common_lib_v0.0.1+': Invalid argument

[generateContainer] Something went wrong. Aborting Container Generation
× fatal: cannot create directory at 'plugins/ern_v0.10.2+/@apps:our_common_lib_v0.0.1+': Invalid argument

runCauldronContainerGen failed: Error: fatal: cannot create directory at 'plugins/ern_v0.10.2+/@apps:our_common_lib_v0.0.1+': Invalid argument

An error occurred: fatal: cannot create directory at 'plugins/ern_v0.10.2+/@apps:rour_common_lib_v0.0.1+': Invalid argument

Basically the issue is the colon in the directory name, which is an illegal character in Windows in directory names. It is caused by our project setup having a custom @apps scope for a shared library (coming from our internal NPM Registry), which for some reason shows up as @apps: in the name. The container builds fine on a Mac (where the colon is legal char). Any thoughts?

**Comment by belemaire** [comment]
@Jeopardye Thanks for reporting and for the details
This is definitely a bug. Colon character is not a reserved one on Linux / Mac for paths, while it is on Windows indeed. FYI Scoped packaged are supported by Electrode Native. The bug affects Electrode Native Windows only :(
Will look into it, bug fix will be included in v0.12.0 along with other fixes that will improve Windows support.
